app/views/user_mailer/invitation.html.erb

```diff
@@ -12,15 +12,19 @@
   </p>
 
   <p>
-    <%= t ".part1" %>
+    <%= t(".html.ao3_intro", otw_link: style_link("Organization for Transformative Works", "http://transformativeworks.org")).html_safe %>
```

Comment:
The org name in `style_link("Organization for Transformative Works", "http://transformativeworks.org")` also needs to be translatable. Also, I believe the URL should use the `https` protocol.